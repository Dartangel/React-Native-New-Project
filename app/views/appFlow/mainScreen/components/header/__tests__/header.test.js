import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from '../header';

describe('Header', () => {
    test('Header snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<Header />);
        expect(snapshot).toMatchSnapshot();
    });
});