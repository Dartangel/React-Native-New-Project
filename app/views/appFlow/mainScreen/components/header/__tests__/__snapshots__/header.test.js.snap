// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header snapshot 1`] = `
<View
  style={
    Object {
      "flex": 0.05,
      "flexDirection": "row",
      "marginBottom": 10,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 0.15,
        "justifyContent": "center",
      }
    }
  >
    <IconComponent
      name="flag-variant"
    />
  </View>
  <View
    style={
      Object {
        "flex": 0.7,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#ffffff",
          "fontSize": 15,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 0.15,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#ffffff",
          "fontSize": 15,
        }
      }
    >
      0/7
    </Text>
  </View>
</View>
`;
